documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

```diff
@@ -69,13 +69,18 @@ Because the synchronization is time-based, any switchover by consumers to a pass
 
 NOTE: If you have an application written in Java, you can use the `RemoteClusterUtils.java` utility to synchronize offsets through the application. The utility fetches remote offsets for a consumer group from the `checkpoints` topic. 
 
-== Connectivity checks
+== Deciding when to use the heartbeat connector
 
-`MirrorHeartbeatConnector` emits _heartbeats_ to check connectivity between clusters.
+The heartbeat connector emits heartbeats to check connectivity between source and target Kafka clusters.
+An internal `heartbeat` topic is replicated from the source cluster, which means that the heartbeat connector must be connected to the source cluster. 
+The `heartbeat` topic is located on the target cluster, which allows it to do the following:
 
-An internal `heartbeat` topic is replicated from the source cluster.
-Target clusters use the `heartbeat` topic to check the following:
+* Identify all source clusters it is mirroring data from
+* Verify the liveness and latency of the mirroring process
 
-* The connector managing connectivity between clusters is running
-* The source cluster is available
+This helps to make sure that the process is not stuck or has stopped for any reason. 
+You enable the heartbeat connector by including its configuration in the MirrorMaker 2 deployment configuration.
+However, using the heartbeat connector is not always necessary and can be left out of the MirrorMaker 2 configuration. 
+For example, if your deployment has low network latency or a small number of topics, you might prefer to monitor the mirroring process using log messages or other monitoring tools. 
+If heartbeats are enabled, you can set the `emit.heartbeats.enabled` property of the heartbeat connector to `false` if you do not want to use them.
```

Comment:
Shouldn't suggest to just removing the connector from the `KafkaMirrorMaker2` instead of setting some disable option? That would also correspond to the examples.

Reply:
Yes. I've updated